cmd/role/role.go

```diff
@@ -186,8 +186,14 @@ var createRoleCmd = &cobra.Command{
 		permissionsMap := map[string][]string{}
 
 		permissionsList, _ := cmd.Flags().GetStringArray("permissions")
-
 		CreatePermissionsMap(permissionsList, permissionsMap)
+		if authApi.ContainsSensitivePermissions(permissionsMap) {
+			viper.BindPFlag("force", cmd.Flags().Lookup("force"))
+			err := util.ConfirmCommand(fmt.Sprintf("Some of the permissions assigned to this role have security implications (such as user, API key, and role management operations). Are you sure you want to proceed with role creation?"), viper.GetBool("force"))
```

Comment:
Why you don't specify the role in this case ? What the point of using `Sprintf` ?

Should not be `Some of the permissions of the role '%s' have security implications (such as user, API key, and role management operations).`